server/tserver/src/main/java/org/apache/accumulo/tserver/tablet/Tablet.java

```diff
@@ -484,10 +485,11 @@ public void sessionExpired() {
         }
 
       } catch (Throwable t) {
+        String msg = "Error recovering tablet " + extent + " from log files";
```

Comment:
Need to be careful 'extent' doesn't contain `{}` here.

Reply:
If I change the else below to just re-throw the throwable, then I can use the sl4j format arg so we don't have to worry about this.